libstring/libstring_test.go

```diff
@@ -169,6 +169,7 @@ func TestCanonicalizeIP(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		tt := tt
```

Comment:
Question, why is this needed?

Reply:
[Here](https://eli.thegreenplace.net/2019/go-internals-capturing-loop-variables-in-closures/) and [here](https://stackoverflow.com/a/68559720/961092) is some information about issue golangci-lint reported.